tests/waku_discv5/test_waku_discv5.nim

```diff
@@ -0,0 +1,426 @@
+{.used.}
```

Comment:
This file seems to have fewer lines than the original `tests/test_waku_discv5.nim`

Reply:
That's correct, because I abstracted some repeated code into functions.